packages/spec/src/index.ts

```diff
@@ -1,18 +1,30 @@
-export declare namespace v1 {
+/**
+ * The Standard Schema interface.
+ *
+ * Root-level alias of StandardSchemaV1.Schema.
+ */
+export type StandardSchemaV1<
+  Input = unknown,
+  Output = Input,
+> = StandardSchemaV1.Schema<Input, Output>;
+
+export declare namespace StandardSchemaV1 {
   /**
    * The Standard Schema interface.
+   *
+   * Equivalent to the root-level StandardSchemaV1 type.
    */
-  interface StandardSchema<Input = unknown, Output = Input> {
+  export interface Schema<Input = unknown, Output = Input> {
```

Comment:
Are the `export` statements inside of `export declare namespace ...` required?

Reply:
Yeah, it gives us more control over what we expose although technically it seems we need to add `export {}` to the namespace to actually get it to behave that way.